src/gui/ControllerRackView.cpp

```diff
@@ -132,6 +133,33 @@ void ControllerRackView::deleteController( ControllerView * _view )
 	song->removeController( c );
 }
 
+void ControllerRackView::moveUp(ControllerView* view)
+{
+	// cannot delete the first module
```

Comment:
It seems like this comment does not belong here in the context of moving things. Guess it should be somthing like "Cannot move the first module further up" or similar.

Reply:
fixed with https://github.com/LMMS/lmms/pull/7139/commits/3d49888b825a2b0d1986448afabd6e6c4b3d4755